### PR TITLE
More updates to live preview

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph 

- reverts the required question styling in nimbus
- updates the required question to mimic mobile
required-action

- adds formnav support in nimbus if we ever go into OQPS mode there

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js [hunk 1]
```diff
@@ -223,6 +228,25 @@ function Form(json) {
...
     });
 
+    self.forceRequiredVisible = ko.observable(false);
+
+    self.showRequiredNotice = ko.computed(function () {
+        return !self.isCurrentRequiredSatisfied() && self.forceRequiredVisible();
+    });
+
+    self.clickedNextOnRequired = function () {
+        console.log('force next');
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js]: Unexpected console statement. (no-console)

Comment (biyeun) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js]: whoops

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js [hunk 2]
```diff
@@ -205,6 +205,11 @@ function Form(json) {
                 || q.answer() !== null;
         });
     });
+    self.isCurrentRequiredSatisfied.subscribe(function (isSatisfied) {
+        if (isSatisfied) {
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js]: 😌

**Review by benrudolph — APPROVED** [review]
awesome. did you run this at all on old cloudcare? i'm a little worried we're hacking up the old ui. my guess is this'll work fine though

**Comment by biyeun** [comment]
@benrudolph all set on the old cloudcare side

**Comment by benrudolph** [comment]
sweet, feel free to merge on ✅

**Pull request merged**